src/app/scheduling/calendar/calendar-sample-3/calendar-sample-3.component.scss

```
@@ -1,10 +1,9 @@
-.sample-wrapper {
-    overflow-y: auto;
-}
-
 .calendar-wrapper {
     max-width: 700px;
     min-width: 200px;
-    max-height: 450px;
-    text-align: center;
-}
\ No newline at end of file
+    margin: 8px;
+}
+
+.igx-calendar{
+   box-shadow: 0 1px 3px 0 rgba(0,0,0,.26), 0 1px 1px 0 rgba(0,0,0,.12), 0 2px 1px -1px rgba(0,0,0,.08);
```

```suggestion
   box-shadow: 0 1px 3px 0 rgba(0,0,0,.26), 
   0 1px 1px 0 rgba(0,0,0,.12), 
   0 2px 1px -1px rgba(0,0,0,.08);
```

Try to keep lines short. The max should be between 80-120 characters.